src/extension/netcdf-out/src/main/java/org/geoserver/wcs/responses/NetCDFCRSWriter.java

```diff
@@ -291,7 +291,8 @@ private void setGridMappingVariableAttributes(NetcdfFileWriter writer, Coordinat
             Set<String> keySet = referencingToNetCDFParameters.keySet();
 
             // getting the list of parameters from the GT Referencing Projection 
-            ParameterValueGroup values = conversionFromBase.getParameterValues();
+            ParameterValueGroup values = projection
+                    .getNetcdfParameters(conversionFromBase.getParameterValues());
```

Comment:
This is the new method implemented in https://github.com/geotools/geotools/pull/1205 to compute NetCDF parameter values from OGC parameter values. This method is required because the rotated pole projection breaks the assumption that parameter mapping is a simple renaming.